src/components/transactions/Switch/SwitchModalContent.tsx

```diff
@@ -0,0 +1,215 @@
+import { normalize, normalizeBN } from '@aave/math-utils';
+import { Trans } from '@lingui/macro';
+import { Box, CircularProgress, Typography } from '@mui/material';
+import React, { useState } from 'react';
+import { FormattedNumber } from 'src/components/primitives/FormattedNumber';
+import { Row } from 'src/components/primitives/Row';
+import { ConnectWalletButton } from 'src/components/WalletConnection/ConnectWalletButton';
+import { useParaswapSellRates } from 'src/hooks/paraswap/useParaswapRates';
+import { useIsWrongNetwork } from 'src/hooks/useIsWrongNetwork';
+import { useModalContext } from 'src/hooks/useModal';
+import { useWeb3Context } from 'src/libs/hooks/useWeb3Context';
+import { useRootStore } from 'src/store/root';
+import { getNetworkConfig, NetworkConfig } from 'src/utils/marketsAndNetworksConfig';
+import { GENERAL } from 'src/utils/mixPanelEvents';
+
+import { AssetInput } from '../AssetInput';
+import { TxModalDetails } from '../FlowCommons/TxModalDetails';
+import { TxModalTitle } from '../FlowCommons/TxModalTitle';
+import { ChangeNetworkWarning } from '../Warnings/ChangeNetworkWarning';
+import { ParaswapErrorDisplay } from '../Warnings/ParaswapErrorDisplay';
+import { SupportedNetworkWithChainId } from './common';
+import { NetworkSelector } from './NetworkSelector';
+import { SwitchActions } from './SwitchActions';
+import { SwitchErrors } from './SwitchErrors';
+import { ReserveWithBalance } from './SwitchModal';
+import { SwitchRates } from './SwitchRates';
+import { SwitchSlippageSelector } from './SwitchSlippageSelector';
+import { SwitchTxSuccessView } from './SwitchTxSuccessView';
+
+interface SwitchModalContentProps {
+  selectedChainId: number;
+  setSelectedChainId: (value: number) => void;
+  supportedNetworks: SupportedNetworkWithChainId[];
+  reserves: ReserveWithBalance[];
+  selectedNetworkConfig: NetworkConfig;
+}
+
+export const SwitchModalContent = ({
+  supportedNetworks,
+  selectedChainId,
+  setSelectedChainId,
+  reserves,
+  selectedNetworkConfig,
+}: SwitchModalContentProps) => {
+  const [slippage, setSlippage] = useState('0.001');
+  const [inputAmount, setInputAmount] = useState('0');
```

Comment:
With amount inputs we usually default to empty string, then the placeholder value will be shown.

Might have to adjust the paraswap hook to not fetch if the amount is an empty string.